tangelo/linq/target/target_cirq.py

```diff
@@ -85,7 +92,40 @@ def simulate_circuit(self, source_circuit: Circuit, return_statevector=False, in
 
             frequencies = {k: v / self.n_shots for k, v in Counter(samples).items()}
         # Noiseless simulation using the statevector simulator otherwise
+        # Run all shots at once and post-process to return measured frequencies on qubits only
```

Comment:
Let's skip a line between the end of a major code branch and the comments for the next major one, for readability

Reply:
ok